projects/core/src/occ/adapters/site-context/default-occ-site-context-config.ts

```diff
@@ -15,8 +15,7 @@ export const defaultOccSiteContextConfig: OccConfig = {
         countries: 'countries',
         regions:
           'countries/${isoCode}/regions?fields=regions(name,isocode,isocodeShort)',
-        baseSites:
-          'basesites?fields=baseSites(uid,defaultLanguage(isocode),urlEncodingAttributes,urlPatterns,stores(currencies(isocode),defaultCurrency(isocode),languages(isocode),defaultLanguage(isocode)),theme,defaultPreviewCatalogId,defaultPreviewCategoryCode,defaultPreviewProductCode,requiresAuthentication)',
+        baseSites: 'basesites?fields=FULL',
```

Comment:
don't forget to create a remind ticket to revert it back in future.

Reply:
Done!